applications/cogip2019-cortex/Makefile

```diff
@@ -18,6 +18,6 @@ FEATURES_REQUIRED += periph_i2c
 USEPKG += vl53l0x-api
 
 # Main thread stack size
-CFLAGS += -DTHREAD_STACKSIZE_MAIN=4096
+CFLAGS += -DTHREAD_STACKSIZE_MAIN=4096 -DSD21_SERVO_POS_NUMOF=3
```

Comment:
Maybe should be great to add a comment in sd21.h to tell that this NUMOF is in the makefile and not in the code.

Reply:
done